src/components/shared/MarkerTooltipContents.js

```diff
@@ -11,6 +11,19 @@ import formatTimeLength from '../../utils/format-time-length';
 import type { TracingMarker } from '../../types/profile-derived';
 import type { MarkerPayload } from '../../types/markers';
 
+function markerDetail(key, label, value) {
```

Comment:
Yes, total_zones is the total in the runtime: http://searchfox.org/mozilla-central/source/js/src/jsgc.cpp#6880

I would think that if all other values have units, then counts would be clear. But yeah, I'm ok either way.